charts/shoot-core/components/charts/coredns/templates/coredns-configmap.yaml

```diff
@@ -15,6 +15,7 @@ data:
         }{{ end }}
       {{- end }}
     }
+     import custom/*.server
```

Comment:
```suggestion
    import custom/*.server
```

Reply:
it does not matter with import, but I modified it.